packages/feature-toggles/src/context/feature-context.tsx

```diff
@@ -0,0 +1,10 @@
+import { createContext } from 'react';
+import IFeature from '../models/feature';
+
+export interface IFeatureContext {
+    getFeature(feature: string): Promise<IFeature>;
+}
+
+export default createContext<IFeatureContext>({
+    getFeature: /* istanbul ignore next */ () => Promise.resolve({}),
+});
```

Comment:
Could you update this change to switch from tabs to spaces?

Reply:
Pretty sure this is all 4 space indents? There's no tabs in this file.

Reply:
![image](https://github.com/awslabs/iot-app-kit/assets/152904116/3cbfa1d0-482d-4fd5-b54c-6f478d08eba1)

Reply:
I see the issue, I don't have eslint hooked up correctly in this new package. Will change.

Reply:
Added eslint, and autofixed these.